layouts/releases/single.html

```diff
@@ -86,19 +86,40 @@ <h3 class="mb-3">Just the source code</h3>
                     </div>
                 </div>
             </li>
-
             <li class="list-group-item p-3">
                 <div class="col">
                     <div class="row">
                         <h3 class="mb-3">Docker</h3>
                     </div>
-                    <div class="row">
-                        <p>
-                            Check out our docker repository on&nbsp;<a
-                                href="https://quay.io/repository/hedgedoc/hedgedoc?tab=tags" target="_blank"
-                                rel="noopener noreferer">quay.io</a>
-                        </p>
+                    <p>
+                        Our docker images are located on quay.io at <a href="https://quay.io/repository/hedgedoc/hedgedoc">hedgedoc/hedgedoc</a>.
+                    </p>
+                    {{ if eq .Page.Params.docker nil }}
+                    <p>
+                        This version was released before the rebranding to HedgeDoc. Docker images can be found in the old repository:
```

Comment:
TBH I'm not happy with this fallback.
Imagine: We release a new version, but the change in the container repository isn't done yet. Therefore the version won't be found on quay and you get this message. :/ Can we find another solution for this?

Reply:
How about a boolean in the docker object that indicates a legacy docker image?

Reply:
How should this legacy parameter be defined? Version comparison for releases prior to 1.7.0?

Reply:
How about "Release name contains codimd"?

Reply:
Done